controller/index.js

```diff
@@ -2,7 +2,7 @@ const scaffold = require('../lib/scaffold');
 const package = require('../lib/package');
 
 const userDependencies = {
-  "password": "git+https://github.com/Prismatik/password.git"
+  "simple-password": "^1.0.1"
```

Comment:
It's here because it's only actually needed if you scaffold a user controller. It's not used in any other controller. So if someone scaffolds a project without users and we had it in the package.mustache it would be an unused dependency that they'd need to remove. It's a bit gross but I think better this way.